cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -122,6 +124,9 @@ macro(configure_sdk_darwin
   set(SWIFT_SDK_${prefix}_OBJECT_FORMAT "MACHO")
 
   foreach(arch ${architectures})
+    # On Darwin, all archs share the same SDK path.
+    set(SWIFT_SDK_${prefix}_ARCH_${arch}_PATH "${SWIFT_SDK_${prefix}_PATH}")
```

Comment:
Having SWIFT_SDK_${prefix}_PATH as only variable works for Darwin since the SDK has all the arches compiled and lipo'd together but other platforms have arch specific sysroot paths. This assumption was limiting other SDKs to only supporting one arch at time. I fixed it here. For Darwin each arch specific SDK path actually just points to the same path.